### PR TITLE
Transfer to Admin (LETA) Validations + code cleanup

### PR DESCRIPTION
**Pull request by dimak1**
*Issue #:* 
- bcgov/entity#15808

*Description of changes:*
- LETA Validations, same as for WILL Transfers
- New HomeOwnersGroupError component for group errors
- Some refactoring, renaming, cleanup
- Add unit tests 

Note: more refactor for validation will be in another PR

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the PPR license (Apache 2.0).

### CODE REVIEW

**Comment by pwei1018** [comment]
Temporary Url for review: https://bcregistry-assets-dev--pr-1323-1r6gitwo.web.app

**Comment by pwei1018** [comment]
Temporary Url for review: https://bcregistry-assets-dev--pr-1323-1r6gitwo.web.app

File: ppr-ui/src/components/mhrRegistration/HomeOwners/HomeOwnersTable.vue [hunk 1]
```diff
@@ -567,11 +538,23 @@ export default defineComponent({
...
+
+        return hasRemovedOwners && isInvalid
+      }
+
+      if (isTransferToExecutorProbateWill.value && props.validateTransfer) {
+        const hasRemovedOwners = TransToExec.hasSomeOwnersRemoved(groupId)
+        const hasExecutors = TransToAdmin.hasAddedAdministratorsInGroup(groupId)
+        const hasRemovedAllOwners = TransToExec.hasAllCurrentOwnersRemoved(groupId)
+        const hasValidDocs = TransToExec.hasOwnersWithValidSupportDocs(groupId)
+        const hasOwnersWithoutDeathCert = !TransToExec.isAllGroupOwnersWithDeathCerts(groupId)
```
Comment (cameron-eyds) [inline ppr-ui/src/components/mhrRegistration/HomeOwners/HomeOwnersTable.vue]: Several of these appear identical in definition, could they be defined once vs defined in each condition perhaps?

**Review by cameron-eyds — APPROVED** [review]
LGTM! 
Just 1 small comment 👍

File: ppr-ui/src/components/mhrRegistration/HomeOwners/HomeOwnersTable.vue (hunk 1 above)
Comment (dimak1) [reply ppr-ui/src/components/mhrRegistration/HomeOwners/HomeOwnersTable.vue]: Yea, I agree, more refactoring is needed. It will be in one of upcoming PRs.

**Review by doug-lovett — APPROVED** [review]
Looks great.

**Review by orelbn — APPROVED** [review]
Looks great, nice refactoring!

**Comment by pwei1018** [comment]
Temporary Url for review: https://bcregistry-assets-dev--pr-1323-1r6gitwo.web.app

**Comment by pwei1018** [comment]
Temporary Url for review: https://bcregistry-assets-dev--pr-1323-1r6gitwo.web.app

**Pull request merged**